src/GameSlice.js

```diff
@@ -1,87 +1,112 @@
-import { createSlice } from '@reduxjs/toolkit'
+import { createSlice } from '@reduxjs/toolkit';
+import { FALLBACK_EQUATION } from './util/constants/fallbackEquation';
+import { Formatter } from './util/Formatter';
+
+const defaultState = {
+  playedToday: false,
+  avgTimeSeconds: 0,
+  kryptoId: null,
+  numUsedObj: {},
+  numbersToUse: [],
+  target: '',
+  validSolutions: [],
+  pageLoaded: false,
+  equation: '',
+  errorMessage: '',
+  solveStreak: 0,
+  currentSeconds: 0,
+  isConfettiOn: false,
+};
+
+function loadInitialState() {
```

Comment:
The initial state form the store now takes whatever is in local storage and adds it on top of the default state object on store init.